main.rb

```diff
@@ -0,0 +1,64 @@
+def decode_char(code)
+  symbols = {
+    '.-': 'A',
+    '-...': 'B',
+    '-.-.': 'C',
+    '-..': 'D',
+    '.': 'E',
+    '..-.': 'F',
+    '--.': 'G',
+    '....': 'H',
+    '..': 'I',
+    '.---': 'J',
+    '-.-': 'K',
+    '.-..': 'L',
+    '--': 'M',
+    '-.': 'N',
+    '---': 'O',
+    '.--.': 'P',
+    '--.-': 'Q',
+    '.-.': 'R',
+    '...': 'S',
+    '-': 'T',
+    '..-': 'U',
+    '...-': 'V',
+    '.--': 'W',
+    '-..-': 'X',
+    '-.--': 'Y',
+    '--..': 'Z',
+    '.----': '1',
+    '..---': '2',
+    '...--': '3',
+    '....-': '4',
+    '.....': '5',
+    '-....': '6',
+    '--...': '7',
+    '---..': '8',
+    '----.': '9',
+    '-----': '0'
+  }
+
+  symbols[code.to_sym]
+end
+
+def decode_word(ciphered)
+  word = ''
+  ciphered.split(' ').each do |code|
+    word += decode_char(code)
+  end
+  word
+end
+
+def decrypt(ciphered)
+  sentence = ''
+  ciphered.split('  ').each do |word|
+    # sentence += decode_word(word) + ' '
+    sentence += "#{decode_word(word)} "
+  end
+  puts sentence
+
+  puts sentence
```

Comment:
- [ ] [Optional] Kindly remove the `puts sentence` in line 60 as it is already written in line 58.